Sources/VaporToolbox/New/New.swift

```diff
@@ -96,7 +96,7 @@ struct New: AnyCommand {
             "Project " + name.consoleText(.info) + " has been created!",
             "",
             "Use " + "cd \(Process.shell.escapeshellarg(cdInstruction))".consoleText(.info) + " to enter the project directory",
-            "Use " + "\(CommandLine.arguments[0]) xcode".consoleText(.info) + " to open the project in Xcode",
+            "Use " + "open \(Process.shell.escapeshellarg(cdInstruction))/Package.swift".consoleText(.info) + " to open the project in Xcode",
```

Comment:
I think we can probably just use `open Package.swift` instead of grabbing the location. That matches the previous behaviour but the suggestion clarifies it
```suggestion
            "Then use " + "open Package.swift".consoleText(.info) + " to open the project in Xcode",
```